go/client/cmd_ctl_autostart.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// +build !darwin
+
+package client
+
+import (
+	"fmt"
+
+	"github.com/keybase/cli"
+	"github.com/keybase/client/go/install"
+	"github.com/keybase/client/go/libcmdline"
+	"github.com/keybase/client/go/libkb"
+)
+
+type CmdCtlAutostart struct {
+	libkb.Contextified
+	ToggleOn bool
+}
+
+func NewCmdCtlAutostart(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command {
+    const backtick = "`"
```

Comment:
```suggestion
	const backtick = "`"
```
^ this looks like spaces instead of a tab

Reply:
Fixed.